packages/ngx-notifications/src/lib/components/notification/notification.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="m-alert" [style.display]="state.value === 'inactive' ? 'none' : ''">
+  <button class="a-button-transparent has-icon m-alert__close" *ngIf="options.closeButton" (click)="remove()"><span class="fa fa-close" aria-label="Close"></span></button>
+  <h4 class="h5 u-margin-bottom-xs" *ngIf="title" [class]="options.titleClass" [attr.aria-label]="title">
```

Comment:
Hier zou ik `<h1>` hanteren voor WCAG. Het ARIA-label mag je weglaten, want is dubbelop.